app/helpers/application_helper.rb

```diff
@@ -16,6 +16,10 @@ def admin_or_moderator?
     access_denied
   end
 
+  def user_not_banned?
```

Comment:
what I want is to use "positive" methods ... it's really simplifying code reading 
```
def active?
  !banned?
end
```

and then you can use `if current_user.active?` where you want